# Number_Divider
More error handling practice. This time with 1 try and 2 catch block statements!
